### PR TITLE
M365 CLI - Bulk library generation

### LINKED ISSUE #274: CLI for M365 version for script "Bulk library generation"
🎯Aim
The aim of this issue would be to add a [CLI for M365](https://pnp.github.io/cli-microsoft365/) version of script [Bulk library generation](https://pnp.github.io/script-samples/spo-create-bulk-libraries/README.html?tabs=pnpps).

🤔How
I think one of the best approach would be to use the following commands:
- [field-add](https://pnp.github.io/cli-microsoft365/cmd/spo/field/field-add/)
- [list-add](https://pnp.github.io/cli-microsoft365/cmd/spo/list/list-add/)
- [list-view-add](https://pnp.github.io/cli-microsoft365/cmd/spo/list/list-view-add/)

### PR DESCRIPTION
**Pull request by milanholemans**
Closes #274 

## Remarks
* `m365 spo list view add` command is in preview at this moment. You should use v5.3.0 preview version of M365 CLI
* Replaced the `TargetSitePartUrl` and the `PartTenant` parameters by `TargetWebUrl`. Right now we assume that the target SharePoint URL is `"https://$($PartTenant).sharepoint.com/sites/$($TargetSitePartUrl)"`. This is not always the case, tenants are able to use other prefixes than `/sites`.

### CODE REVIEW

File: scripts/spo-create-bulk-libraries/README.md [hunk 1]
```diff
@@ -87,6 +83,66 @@ process {
 [!INCLUDE [More about PnP PowerShell](../../docfx/includes/MORE-PNPPS.md)]
 ***
 
+# [CLI for Microsoft 365 with PowerShell](#tab/cli-m365-ps)
+
+```powershell
+# Usage example: 
+#     .\Create-Bulk-Libraries.ps1 -TargetWebUrl https://contoso.sharepoint.com/sites/Intranet AmountLibraries 3
```
Comment (Adam-it) [inline scripts/spo-create-bulk-libraries/README.md:90]: I think in the example comment we are missing the `-` 🤔. Should be `-AmountLibraries`

**Review by Adam-it — COMMENTED** [review]
pulled locally and tested ✅
works like a charm 👍 
thanks for the additional improvements 👍

File: scripts/spo-create-bulk-libraries/README.md (hunk 1 above)
Comment (milanholemans) [reply scripts/spo-create-bulk-libraries/README.md:90]: Aw yeah, good catch. Let me fix that!

**Comment by pkbullock** [comment]
Thank you @milanholemans 👏 for taking time to contribute - it is highly appreciated. This will be live shortly 😊

@Adam-it thank you for testing the sample as well ✨😁

**Comment by pkbullock** [comment]
@milanholemans - Do you have a Twitter Handle for Socialization of the sample? No worries if not.

**Comment by milanholemans** [comment]
> @milanholemans - Do you have a Twitter Handle for Socialization of the sample? No worries if not.

Am one of the few people without Twitter :)

**Pull request merged**